pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -715,6 +729,76 @@ public ApnsPayloadBuilder addCustomProperty(final String key, final Object value
         return this;
     }
 
+    /**
+     * <p>Sets the provided content state object inside content-state payload for Live Activities.</p>
+     *
+     * <p>The precise strategy for serializing the values of custom properties is defined by the specific
+     * {@code ApnsPayloadBuilder} implementation.</p>
+     *
+     * @param contentState content-state object
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
+     *      Updating and ending your Live Activity with remote push notifications</a>
+     */
+    public ApnsPayloadBuilder setContentState(final Map<String, Object> contentState) {
+        this.contentState = contentState;
+        return this;
+    }
+
+    /**
+     * <p>Sets whether the notification payload will be used for updating the Live Activity
+     * or for ending it.</p>
+     *
+     * @param event the type of update event to send to the destination activity
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
+     *      Updating and ending your Live Activity with remote push notifications</a>
+     */
+    public ApnsPayloadBuilder setEvent(final LiveActivityEvent event) {
+        this.event = event;
+        return this;
+    }
+
+    /**
+     * <p>Sets a timestamp for the push notification payload.   </p>
+     *
+     * @param timestamp Timestamp
```

Comment:
Can we say more about this? I guess this is related to the other question, though, in that I guess we need to find some explanation for what this really does before we can explain it to Pushy's end users.